src/home_robot_hw/home_robot_hw/stretch_client/modules/head.py

```diff
@@ -64,15 +64,15 @@ def set_pan_tilt(
         if blocking:
             self.wait()
 
-    def look_at_ee(self, blocking: bool = False):
+    def look_at_ee(self, blocking: bool = True):
         pan, tilt = self._robot_model.look_at_ee
         self.set_pan_tilt(pan, tilt, blocking=blocking)
 
-    def look_front(self, blocking: bool = False):
+    def look_front(self, blocking: bool = True):
         pan, tilt = self._robot_model.look_front
         self.set_pan_tilt(pan, tilt, blocking=blocking)
 
-    def look_ahead(self, blocking: bool = False):
+    def look_ahead(self, blocking: bool = True):
```

Comment:
maybe comments to differntiate - look ahead i think is straight ahead, look front is angled down 45 degrees (navigation posture)